Formula/go.rb

```diff
@@ -7,11 +7,11 @@ class Go < Formula
     if Hardware::CPU.arm?
       url "https://golang.org/dl/go1.16beta1.src.tar.gz"
       sha256 "48e032c8cf71af4dc8119a29ee829c4fbd5265e32fd012564d4a70bb207695c1"
-      version "1.15.6"
```

Comment:
This looks quite sketchy, what about `1.16beta1` instead?

Reply:
That won't work, and will prevent any bottles from being provided on ARM.

You actually need to change this to 1.15.7.

Reply:
That's exactly what I did already :)